starter_code/CSS/styles.css

```
@@ -0,0 +1,170 @@
+header {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    background-color: black;
+    height: 100px;
+    border-bottom: yellow solid 3px;
+  }
+  .logo {
+    color: yellow;
+    font-size: 40px;
+    padding: 10px;
+  }
+  nav {
+    
+    align-self: flex-end;
+    padding: 10px;
+    display: flex;
+    justify-content: space-around;
+  }
+  nav a {
+    color: white;
+    font-size: 25px;
+    text-decoration: none;
+    padding: 10px;
+  }
+  .are {
+    text-decoration: underline;
+    text-decoration-color:yellow;
+  }
+  .taste {
+    text-decoration: line-through;
+    color: gray;
+    text-align: center;
+  }
+  main {
+    
+    background-image: url('../images/coworking.jpg');
+    height: 700px;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+  section {
+    height: 50px;
+    background-color: lightgray;
+    font-size: 30px;
+    text-align: center;
+    padding-top: 10px;
+  }
+  .inner-box{
+    
+    position:absolute;
+    background:rgba(255,255,255,0.7);
+    overflow: scroll;
+    top:70%;
+    left:85%;
+    width: 250px;
+    transform:translate(-50%,-50%);
+    padding:35px;
+    text-align:center;
+    }
+
+    .inner-box h2{
+    font-family:sans-serif;
+    text-align:center;
+    font-size:26px;
+    font-weight:800;
+    color:black;
+    margin: 0%;
+    }
+    .inner-box p{
+    font-family:sans-serif;
+    text-align:center;
+    font-size:18px;
+    font-weight:500;
+    color:rgb(0, 0, 0);
+    padding: 5px;
+    line-height: 1.6;
+    }
+    .inner-box button {
+    background-color: yellow;
+    border: none;
+    color:black;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    font-weight: 800;
+    margin: 4px 2px;
+    cursor: pointer;
+    }
+    .base-container {
+    display: grid;
+    margin: 10px;
+    }
+    .grid04 {
+    grid-template-columns: 34% 33% 33%;
+    }
+    .base-child {
+    background-color: white;
+    border-right: 1px solid rgb(194, 191, 191);
+    }
+    .base-child h3{
```

`.base-child-1 h3` selector has the same style, so you can write it here
```suggestion
    .base-child h3, .base-child-1 h3{
```